pkgs/by-name/wl/wl-clicker/package.nix

```
@@ -18,29 +21,31 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   src = fetchFromGitHub {
-    owner = "phonetic112";
+    owner = "phoneticalb";
     repo = "wl-clicker";
-    rev = "v${finalAttrs.version}";
-    hash = "sha256-HuUK1kkqIdB5SdcewJ1J8elzqrFaFWC3BRV3GgcasTU=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-+k3iZOv12WbqpeYbYjIXBIB4mO2DrY1pl+MJn2B+cZA=";
   };
 
-  postPatch = ''
-    sed -i 's|/usr|${wlr-protocols}|g' Makefile
-  '';
+  makeFlags = [ "CC=${lib.getExe cc}" ];
```

I guess

```suggestion
  makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];
```

Should be enough?